.circleci/config.yml

```diff
@@ -100,8 +100,6 @@ commands:
             python -m pip install --user \
                 numpy<< parameters.numpy_version >> \
                 -r requirements/doc/doc-requirements.txt
-            python -m pip install --no-deps --user \
-                git+https://github.com/matplotlib/mpl-sphinx-theme.git
```

Comment:
This should be kept.

Reply:
fixed, this one didn't cause a merge conflict